rholang/src/test/scala/coop/rchain/rholang/Resources.scala

```diff
@@ -60,27 +62,22 @@ object Resources {
       ](context, Branch(branch))
     }
 
-    mkTempDir(prefix)
+    mkTempDir(prefix)(implicitly[Concurrent[F]])
       .flatMap(tmpDir => Resource.make(mkRspace(tmpDir))(_.close()))
   }
 
-  def mkRuntime(
+  def mkRuntime[F[_]: ContextShift: Concurrent: Log: Metrics, M[_]: Parallel[F, ?[_]]](
       prefix: String,
-      storageSize: Long = 1024 * 1024,
-      storeType: StoreType = StoreType.LMDB
-  )(
-      implicit log: Log[Task],
-      scheduler: Scheduler,
-      metrics: Metrics[Task]
-  ): Resource[Task, Runtime[Task]] =
-    mkTempDir[Task](prefix)
+      storageSize: Long = 1024 * 1024
+  )(implicit scheduler: Scheduler): Resource[F, Runtime[F]] =
```

Comment:
I wonder why scheduler is needed. Compiler is not complaining so I guess it's there for a reason.